Content.Client/Verbs/UI/VerbMenuUIController.cs

```diff
@@ -77,6 +77,9 @@ public void OpenVerbMenu(EntityUid target, bool force = false, ContextMenuPopup?
             CurrentVerbs = _verbSystem.GetVerbs(target, user, Verb.VerbTypes, force);
             OpenMenu = menu;
 
+            if (CurrentVerbs.Count == 0)
+                return;
```

Comment:
```suggestion
            if (EntityManager.IsClientSide(target) && CurrentVerbs.Count == 0)
                return;
```
Just cause the client doesn't have client-side verbs doesn't mean that there aren't any server-side ones. So you need to add separate logic for when server-side verbs are received and there still aren't any verbs.

Reply:
if its a clientside entity the server *cant* have any verbs since it doesnt think it exists?

Reply:
yeah hence the suggested client-side entity check. but if its not a client side entity, then there might be server side verbs

Reply:
Not exactly sure why- but making the requested changes makes the issue appear again.

Reply:
> So you need to add separate logic for when server-side verbs are received and there still aren't any verbs.

The "waiting for server" thing should still appear, you just need to make it disappear when the server responds and there are still no verbs.